modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -11,7 +11,7 @@
 // * installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc
 // * installing/installing_openstack/installing-openstack-installer-restricted.adoc
 // * installing/installing_platform_agnostic/installing-platform-agnostic.adoc
-// * installing/installing_rhv/installing-rhv-restricted-network.adoc
+
```

Comment:
Did you intend to have an extra line here?

Reply:
No, I didn't notice that an open line remained there...